Project-1/README.md

```diff
@@ -13,63 +13,226 @@ Note: The standard interpretation of the logical symbols - "∨", "∧", "→",
 
 1. Provide the truth tables for each of the following propositional logic formulas. State whether each is a tautology, a contradiction, or contingent:
   ```
-  (a) (¬A→B)∨((A∧¬C)→B) 
-  (b) (A→B)∧(A→¬B)
-  (c) (A→(B∨C))∨(C→¬A) 
-  (d) ((A→B)∧C)∨(A∧D) 
+  (a) (¬A→B)∨((A∧¬C)→B) Tautology
+  A	B	C	((¬A → B) ∨ ((A ∧ ¬C) → B))
+F	F	F	T
+F	F	T	T
+F	T	F	T
+F	T	T	T
+T	F	F	T
+T	F	T	T
+T	T	F	T
+T	T	T	T
+
+  (b) (A→B)∧(A→¬B) Contingent
+  A	B	((A → B) ∧ (A → ¬B))
+F	F	T
+F	T	T
+T	F	F
+T	T	F
+
+  (c) (A→(B∨C))∨(C→¬A) Tautology
+  A	B	C	((A → (B ∨ C)) ∨ (C → ¬A))
+F	F	F	T
+F	F	T	T
+F	T	F	T
+F	T	T	T
+T	F	F	T
+T	F	T	T
+T	T	F	T
+T	T	T	T
+  (d) ((A→B)∧C)∨(A∧D) Contingent
+  A	B	C	D	(((A → B) ∧ C) ∨ (A ∧ D))
+F	F	F	F	F
+F	F	F	T	F
+F	F	T	F	T
+F	F	T	T	T
+F	T	F	F	F
+F	T	F	T	F
+F	T	T	F	T
+F	T	T	T	T
+T	F	F	F	F
+T	F	F	T	T
+T	F	T	F	F
+T	F	T	T	T
+T	T	F	F	F
+T	T	F	T	T
+T	T	T	F	T
+T	T	T	T	T
   ```
 	
 2. A _literal_ is an atomic formula or the negation of an atomic formula. We say a formula is in _conjunctive normal form_ (CNF) if it is the conjunction of the disjunction of literals. Find propositional logic formulas in CNF equivalent to each of the following:
   ```
   (a) (A→B)→C
+  (A∨C)∧(¬B∨C)
   (b) (A→(B∨C))∨(C→¬A)
+  Av~A
   (c) (¬A∧¬B∧C)∨(¬A∧¬C)∨(B∧C)∨A 
+ Bv~B
```

Comment:
I used different letters solely for the sake of variety there. Since both are tautologies, any formula of the form "Pv~P" will be correct so I could have said Av~A or Bv~B and both would be correct.